quickwit/quickwit-storage/Cargo.toml

```diff
@@ -70,11 +74,19 @@ azure = [
   "azure_storage/enable_reqwest_rustls",
   "azure_storage_blobs/enable_reqwest_rustls",
 ]
+google = [
```

Comment:
can we call the feature gfs instead of google?

Reply:
Sure, I will update this place tomorrow. 😋

Reply:
i ll do it actually... i spotted a bunch of other problems.